File: BackEnd/DashDelivery/src/main/java/com/dashdelivery/DashDelivery/Models/Pedido.java
package com.dashdelivery.DashDelivery.Models;

import jakarta.persistence.*;
import lombok.Data;

import java.util.Date;


@Data
@Entity
public class Pedido {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nomeCliente;
    private String enderecoCliente;

    @Lob
    private String itens;
    private String formaDePagamento;
    private String status;

    private Double total;


}

File: FrontEnd/dashdelivery/src/app/models/Pedido.ts
export interface Pedido {
  id:number;
  nomeCliente: string;
  enderecoCliente: string;
  itens: string;
  total: number;
  status: string;
  formaDePagamento:string;

}


export interface ItemPedido {
  nomeProduto: string;
  quantidade: number;
  preco: number;
}
